launcher/src/main/java/org/wso2/carbon/launcher/CarbonServerEvent.java

```diff
@@ -76,4 +96,17 @@ public int getType() {
     public CarbonLaunchConfig getConfig() {
         return config;
     }
+
+    /**
+     * returns the OSGi system bundle context.
+     * @return system bundle context
+     */
+    public BundleContext getSystemBundleContext() {
+        return systemBundleContext;
+    }
+
+    // Sets the system bundle context for the CarbonServerEvent
+    void setSystemBundleContext(BundleContext systemBundleContext) {
+        this.systemBundleContext = systemBundleContext;
+    }
```

Comment:
@Mirage20 I do not see any usage of getSystemBundleContext() method. Can you please explain the purpose of adding this property? Thanks!